beancount.el

```diff
@@ -259,6 +261,11 @@ from the open directive for the relevant account."
 (defconst beancount-metadata-regexp
   "^\\s-+\\([a-z][A-Za-z0-9_-]+:\\)\\s-+\\(.+\\)")
 
+(defconst beancount-open-directive-regexp
+  (concat "^\\(" beancount-date-regexp "\\) +"
+          "\\(open\\) +"
+          "\\(" beancount-account-regexp "\\) +"))
```

Comment:
This is problematic as it requires the `open` directive to have a space after the account name, but one is not required:
```beancount
2024-06-03 open Assets:Foo
```
is a valid open directive.

Reply:
Ooops. Should be good now.